test/integration/api/services_controller_test.rb

```diff
@@ -311,21 +311,6 @@ def setup
     end
   end
 
-  class ServiceCreateTest < self
```

Comment:
This test is still useful as far as I reckon

Reply:
now only first test is useful, because now we are showing a single flash error message i,e,  **Couldn't create Product. Check your Plan limits**, because we are showing rest errors  as inline and doesn't make sense to show single error twice so added first test and removed last two

Reply:
Removing a test is a risky business and in this case implies a coverage loss. Keep the test but update the assertions.